trafficgen-server-start

```diff
@@ -89,6 +100,13 @@ while [ $# -gt 0 ]; do
         --testpmd-dst-macs)
             testpmd_dst_macs="$val"
             ;;
+        --testpmd-dev-cpu-ratio)
+            # a value N, where N:1 is a ratio of N devices per CPU used for PMD threads
+            # ex:
+            # --testpmd-dev-cpu-ratio 1  :   1 cpu needed for every device
+            # --testpmd-dev-cpu-ratio 4  :   1 cpu needed for every 4 devices
+            testpmd_dev_cpu_ratio="$val"
+            ;;
```

Comment:
Hmm, I was looking to see if this was now a required parameter because I didn't see a default value set and I don't actually see this referenced anywhere else in this patch.  What is going on here?

Reply:
I accidentally left this in. It is not needed with testpmd-queues-per-pmd